### DISCUSSION TITLE
Is it possible to deploy to fly.io ?

### DISCUSSION THREAD

**Discussion by Thenafi** [prisma/prisma#16556] [Q&A]
If so a guide will be excellent.

**Comment by pantharshit00** [comment]
Hello @Thenafi,

It is possible to deploy Prisma to fly.io. I found a pretty old example which you can reference here: https://github.com/janpio/fly-prisma-example. There used to be an issue when trying deployment there but it was resolved: https://github.com/superfly/flyctl/issues/405


Deployment on fly.io should not be different from any other Node.js application for Prisma so you can follow fly's documentation on how you can deploy a Node.js application.

**Reply by vladi-stevanovic** [reply]
👋 Hi @Thenafi, did you perhaps have a chance to check Harshit's recommendation? Let us know if you still have any questions! 

Or, if you've found a solution, we'd love to know how you solved this problem. Feel free to post it in this thread and "Mark as answer" if this discussion is now solved!

**Reply by Thenafi** [reply]
I haven't got the time to use the solution.i wil test it and report here

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
